lib/resources/pip.rb

```diff
@@ -75,7 +81,7 @@ def pip_cmd
           return nil
         end
       end
-      pipcmd || 'pip'
+      @virtualenv_path.nil? ? pipcmd || 'pip' : @virtualenv_path
```

Comment:
So, if `@virtualenv_path` is _nil_,  `@virtualenv_path.nil?` is _true_. However, `@virtualenv_path` will also be understood as "falsy" by ruby, so that line is equivalent to
```ruby
@virtualenv_path || pipcmd || 'pip'
```